Sources/Shared/VimeoClient.swift

```diff
@@ -27,67 +27,62 @@
-
 final public class VimeoClient {
     // MARK: -
     
-    /**
-     *  `RequestToken` stores a reference to an in-flight request
-     */
+    /// `RequestToken` stores a reference to an in-flight request
     public struct RequestToken {
+
         /// The path of the request
         public let path: String?
         
         /// The data task of the request
-        public let task: Cancelable?
-        
-        /**
-         Cancel the request
-         */
+        public let task: Task?
+
+        /// Cancels the request
         public func cancel() {
             self.task?.cancel()
         }
     }
     
-        /// Dictionary containing URL parameters for a request
+    /// Dictionary containing URL parameters for a request
     public typealias RequestParametersDictionary = [String: Any]
     
-        /// Array containing URL parameters for a request
+    /// Array containing URL parameters for a request
     public typealias RequestParametersArray = [Any]
     
-        /// Dictionary containing a JSON response
+    /// Dictionary containing a JSON response
     public typealias ResponseDictionary = [String: Any]
     
-        /// Domain for errors generated by `VimeoClient`
+    /// Domain for errors generated by `VimeoClient`
     public static let ErrorDomain = "VimeoClientErrorDomain"
     
     // MARK: -
     
     /// Session manager handles the http session data tasks and request/response serialization
     public typealias SessionManagingAuthenticationListening = SessionManaging & AuthenticationListeningDelegate
-    fileprivate var sessionManager: SessionManagingAuthenticationListening? = nil
+    private var sessionManager: SessionManagingAuthenticationListening? = nil
     
-        /// response cache handles all memory and disk caching of response dictionaries
+    /// response cache handles all memory and disk caching of response dictionaries
     private let responseCache = ResponseCache()
 
     private var reachabilityManager: ReachabilityManaging?
-    
-    struct Constants {
-        fileprivate static let BearerQuery = "Bearer "
-        fileprivate static let AuthorizationHeader = "Authorization"
-        
-        static let PagingKey = "paging"
-        static let TotalKey = "total"
-        static let PageKey = "page"
-        static let PerPageKey = "per_page"
-        static let NextKey = "next"
-        static let PreviousKey = "previous"
-        static let FirstKey = "first"
-        static let LastKey = "last"
+
+    enum Constants {
+        fileprivate static let bearerQuery = "Bearer "
+        fileprivate static let authorizationHeader = "Authorization"
+
+        static let pagingKey = "paging"
+        static let totalKey = "total"
+        static let pageKey = "page"
+        static let perPageKey = "per_page"
+        static let nextKey = "next"
+        static let previousKey = "previous"
+        static let firstKey = "first"
+        static let lastKey = "last"
     }
```

Comment:
One of the response keys is currently used outside of this file so that stopped me from adding them as private extensions on String. But I agree separating them and moving down the bottom of the file makes sense!